ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -53,6 +53,16 @@ tests:
         TEST_FUNC=test_install
         MAKEFILE_TARGET="create_full_environment run test_parallel"
     workflow: baremetalds-assisted-install
+- as: system-test-periodic
+  cron: '@daily'
+  steps:
+    cluster_profile: packet
+    env:
+      ASSISTED_CONFIG: |
+        OPENSHIFT_VERSION=""
```

Comment:
what will it do?

Reply:
maybe add a comment if it's not totally clear

Reply:
It will run against all the versions instead of a single version. I don't think we can comment in there